extensions/wikia/Email/EmailExceptions.class.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Email;
+
+class ControllerException extends \Exception {
+	public $errorType = 'error';
```

Comment:
Maybe replace with `function getErrorType()`, and derived classes override this method.